lib/utils/caclulateOdds.js

```diff
@@ -0,0 +1,15 @@
+import { ethers } from 'ethers'
+
+export const caclulateOdds = (usersTicketBalance, totalSupply, ticketDecimals) => {
+  if (usersTicketBalance.eq(ethers.utils.bigNumberify(0))) {
+    return 0
+  }
+
+  const usersBalanceFloat = Number(
+    ethers.utils.formatUnits(usersTicketBalance, Number(ticketDecimals))
+  )
+
+  const totalSupplyFloat = Number(ethers.utils.formatUnits(totalSupply, Number(ticketDecimals)))
+
+  return totalSupplyFloat / usersBalanceFloat
```

Comment:
Would be good to update this as well since we changed this today to be divided by the numberOfWinners (default to 1 if num winners isn't set).